.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,15 +1,26 @@
-## Pull Request Guidelines
+## Checklist for submitting a Pull Request (delete this when complete)
 
 1. Please read our [contributor guidelines](https://github.com/kubernetes/kubernetes/blob/master/CONTRIBUTING.md).
-1. See our [developer guide](https://github.com/kubernetes/kubernetes/blob/master/docs/devel/development.md).
-1. Follow the instructions for [labeling and writing a release note for this PR](https://github.com/kubernetes/kubernetes/blob/master/docs/devel/pull-requests.md#release-notes) in the block below.
+2. See our [developer guide](https://github.com/kubernetes/kubernetes/blob/master/docs/devel/development.md).
+3. Follow the instructions for [labeling and writing a release note for this PR](https://github.com/kubernetes/kubernetes/blob/master/docs/devel/pull-requests.md#release-notes) in the block below.
+
 
 ```release-note
-* Use the release-note-* labels to set the release note state 
-* Clear this block to use the PR title as the release note 
+* Use the release-note-* labels to set the release note state
+* Clear this block to use the PR title as the release note
 -OR-
 * Enter your extended release note here
 ```
 
+4. If you want this PR to automatically close an issue when it is merged,
+   add the following to the pull request message:
+
+`fixes #<issue number>`
+
+or to close multiple issues:
+
+`fixes #<issue number>, fixes #<issue number>, fixes #<issue number>`
+
+See: https://github.com/blog/1506-closing-issues-via-pull-requests
```

Comment:
Why not link the text "want this PR to automatically close an issue" to save verbosity on this template.  The less people have to read, the more likely they will.